native/cocos/base/StringUtil.cpp

```diff
@@ -28,7 +28,9 @@
 #include <algorithm>
 #include <cctype>
 #include <cstdarg>
-#include "base/ZipUtils.h"
+#ifndef CC_WGPU_WASM
+    #include "base/ZipUtils.h"
+#endif
```

Comment:
Why change it?

Reply:
this denpends on libz, while wgpu does not.